packages/tools/examples/segmentBidirectionalTool/findLargestBidirectional.ts

```diff
@@ -0,0 +1,218 @@
+import { cache } from '@cornerstonejs/core';
+import { adaptersRT } from '@cornerstonejs/adapters';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+
+import vtkImageMarchingSquares from '@kitware/vtk.js/Filters/General/ImageMarchingSquares';
+import vtkDataArray from '@kitware/vtk.js/Common/Core/DataArray';
+import vtkImageData from '@kitware/vtk.js/Common/DataModel/ImageData';
+import { vec3 } from 'gl-matrix';
+
+const vtkUtils = {
+  vtkImageMarchingSquares,
+  vtkDataArray,
+  vtkImageData,
+};
+
+const EPSILON = 1e-2;
+
+export default function contourAndFindLargestBidirectional(segmentations) {
```

Comment:
please add documentation and types when you move this down into the library

Reply:
If the only problem with not having this in the library is the import of adapters above, we can improve it by refactoring and moving the relevant part from adapters down into tools. We can also add the necessary dependency, which should be fine.